.node-version

```diff
@@ -0,0 +1 @@
+16
```

Comment:
had to pin the node version to 16 since it wouldn't build otherwise: https://stackoverflow.com/a/69699772